build.sh

```diff
@@ -81,6 +81,12 @@ if [ -z "$VERSION" ]; then
     help
 fi
 
+# Check if zip is installed
+if [ zip ]; then
```

Comment:
`[ zip ]` is always true, this should be `if ! command -v zip >/dev/null; then`

Reply:
Hi, @Ordoviz! Thanks for your review. You are right. Fixed in 2e5233655036a83dddf59a6146f0f762315c3904.